app/src/main/java/com/example/parkingsystem/activities/MainActivity.java

```diff
@@ -30,5 +31,10 @@ public void onDialogPositiveClick(int parkingSpaces) {
 
     public void setListener() {
         binding.buttonMainSelectParking.setOnClickListener(view -> presenter.onButtonMainSelectParkingPressed());
+        binding.buttonMainBookParkingLot.setOnClickListener(view -> presenter.onButtonMainBookParkingLotPressed());
+    }
+
+    public void onButtonMainBookParkingLotPressed() {
+        Intent intent = new Intent(this, ParkingSpaceReservationActivity.class);
```

Comment:
you should follow the MVP structure

Reply:
I forgot to delete this method, the MVP structure was already done for this listener

Reply:
Deleted it!